### DISCUSSION TITLE
"Too many open files" in client installed from source, no problem in package installed

### DISCUSSION THREAD

**Discussion by f-utility** [abraunegg/onedrive#2011] [Q&A] [answered]
Hi, I recently installed onedrive (thanks for it!) in two computers, one running Debian 11 (easy installation from rep) and the other one Debian 10 (installed from source). In the first case all is well, but not in the second: when I do `onedrive --monitor` I get `ERROR: inotify_init failed (Too many open files)` . I have not changed anything other than the folder to sync, using the config file in both computers in the same way. Following an internet search I used the command `ulimit -Sn` and the limit is the same in both, 1024. If I do `onedrive --synchronize` it works, but I want it run automatically via systemctl (that's how I found the error). Any ideas on how to fix it are appreciated, thanks.

**Comment by abraunegg** [comment, marked as answer]
@f-utility 
This issue stems from the *default* Debian configuration of the Operating System where the number of open files is different between Debian 10 and Debian 11.

Please re-read the following on how to resolve: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-in-monitor-mode
